### PR TITLE
Clarify the "Wrapping Callback-Based Functions" example

### PR DESCRIPTION
**Pull request by thai-d-v**
Hello everyone!

I suggest a small change to make the example easier to understand. I hope it's matched your vision for this project.

### CODE REVIEW

**Comment by ktoso** [comment]
Hi there, it's not a bad idea to clarify this bit and the "exactly one".

I'd propose we change it a little bit though, keep the `// ... do some work here ...`

but move the "invoke exactly once" out of the code and into a note below it:

> Note: You have to take care to _resume_ the continuation _exactly-once_. Resuming a checked continuation will cause an expected crash, guarding you from undefined behavior.

**Comment by thai-d-v** [comment]
@ktoso Thank you, the note is added.

**Comment by ktoso** [comment]
WDYT @mattmassicotte ? looks ok to me

**Review by ktoso — APPROVED** [review]

File: Guide.docc/IncrementalAdoption.md [hunk 1]
```diff
@@ -38,12 +38,17 @@ _continuations_.
...
         updateStyle(backgroundColor: backgroundColor) {
+            // ... do some work here ...
+
             continuation.resume()
         }
     }
 }
 ```
 
+> Note: You have to take care to _resume_ the continuation _exactly-once_.
```
Comment (hborla) [inline Guide.docc/IncrementalAdoption.md:49]:
```suggestion
> Note: You have to take care to _resume_ the continuation _exactly once_. 
```

File: Guide.docc/IncrementalAdoption.md [hunk 2]
```diff
@@ -38,12 +38,17 @@ _continuations_.
...
+            // ... do some work here ...
+
             continuation.resume()
         }
     }
 }
 ```
 
+> Note: You have to take care to _resume_ the continuation _exactly-once_. 
+> Resuming a checked continuation will cause an expected crash, guarding you from undefined behavior.
```
Comment (hborla) [inline Guide.docc/IncrementalAdoption.md:50]:
```suggestion
> Resuming a checked continuation more than once will cause an expected crash, guarding you from undefined behavior.
```

**Comment by mattmassicotte** [comment]
Yes, great to call this behaviour out. Thanks so much!

**Pull request merged**